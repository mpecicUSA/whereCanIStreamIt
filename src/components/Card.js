import React, { Component } from 'react'

export class Card extends Component {
    render() {
        return (
            <div>
                This is a card
            </div>
        )
    }
}

export default Card
