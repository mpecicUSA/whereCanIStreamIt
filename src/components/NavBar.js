import React, { Component } from 'react'
import Button from './Button.js'
import SearchBar from './SearchBar.js'

const divStyle= {
    display: 'flex', 
    justifyContent: 'center',
    background: 'grey',
    color: 'white', 
}
export class NavBar extends Component {
    
    divStyle= {
        display: 'flex', 
        justifyContent: 'center',
    }
    render() {
        return (
            <div style={divStyle}>
                <SearchBar />
                <Button text="Login" />
                <Button text="Register" />
            </div>
        )
    }
}

export default NavBar
