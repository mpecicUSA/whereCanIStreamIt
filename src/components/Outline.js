import React, { Component } from 'react'

export class Outline extends Component {
    render() {
        return (
            <div>
                This is an outline
            </div>
        )
    }
}

export default Outline
