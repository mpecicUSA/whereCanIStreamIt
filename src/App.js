import React from 'react';
import NavBar from './components/NavBar.js'
import Outline from './components/Outline.js'
function App() {
  return (
    <div className="App">
      <NavBar />
      <Outline />
    </div>
  );
}

export default App;
