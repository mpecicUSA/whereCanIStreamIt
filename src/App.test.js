import React from "react";
import renderer from "react-test-renderer";
import App from "./App";
import NavBar from "./components/NavBar.js";
import Outline from "./components/Outline.js";
import Card from "./components/Card.js";

describe("it should render the app", () => {
  test("renders a App Component", () => {
    const tree = renderer.create(<App />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  test("renders a navBar Component", () => {
    const tree = renderer.create(<NavBar />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  test("renders a Outline Component", () => {
    const tree = renderer.create(<Outline />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  test("renders a Card Component", () => {
    const tree = renderer.create(<Card />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
