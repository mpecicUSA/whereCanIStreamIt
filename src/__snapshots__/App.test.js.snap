// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it should render the app renders a App Component 1`] = `
<div
  className="App"
>
  <div>
    a
  </div>
  <div>
    This is a card
  </div>
</div>
`;

exports[`it should render the app renders a Card Component 1`] = `
<div>
  This is a card
</div>
`;

exports[`it should render the app renders a Outline Component 1`] = `
<div>
  This is an outline
</div>
`;

exports[`it should render the app renders a navBar Component 1`] = `
<div>
  a
</div>
`;
